component/_forms.scss

```diff
@@ -31,11 +32,18 @@ label {}
 		font-style: italic;
 	}
 
-	@include touch-hover() {
+  // Hint the user that this element is interactive
```

Comment:
This read weird to me, what about something like

// A visual hint for the user that this element is interactive